src/cli.ts

```diff
@@ -16,14 +18,74 @@ import _ from 'lodash';
 
 import {
   Launcher,
-  ExitStatus,
+  ExitStatus
+} from './cardanoLauncher';
+
+import {
   ServiceExitStatus,
   serviceExitStatusMessage,
-} from './cardanoLauncher';
+} from './service';
 
 import * as byron from './byron';
 import * as jormungandr from './jormungandr';
 
+const waitForExit = setInterval(function() {}, 3600000);
```

Comment:
All this cli main stuff is obscuring the test cases. It should be in a separate module.

Reply:
Oh I thought this was cli.test.ts -- never mind.

But why is the `cli()` function removed? You can just put a `cli(process.argv);` at the end of the file.

Reply:
I will restore the `cli` function